src/Epr.Reprocessor.Exporter.UI/Views/Home/Index.cshtml

```diff
@@ -1,23 +1,52 @@
-﻿@using Microsoft.AspNetCore.Mvc.TagHelpers
-
+﻿@model Epr.Reprocessor.Exporter.UI.ViewModels.HomeSettings
 @{
     ViewData["Title"] = "Home";
```

Comment:
All the non dynamic content should come from resource file as we need to support welsh translation. Check Stephen PRs on account creation repo for reference